src/patternfly/components/BackgroundImage/background-image.hbs

```diff
@@ -1,3 +1,7 @@
-<div class="pf-c-background-image {{ pf-c-background-image--modifier }}">
+<div class="pf-c-background-image {{#if background-image--modifier}} {{background--image--modifier}}{{/if}}"
```

Comment:
background--image--modifier should be background-image--modifier

Reply:
resolved, thanks

Reply:
Can you remove the space before the `{{#if}}` block, too?